plugins/github-actions/src/components/Cards/RecentWorkflowRunsCard.tsx

```diff
@@ -81,7 +81,9 @@ export const RecentWorkflowRunsCard = ({
         <Button
           variant="contained"
           color="primary"
-          href={`https://github.com/${owner}/${repo}/actions/new`}
+          href={`https://${
```

Comment:
Maybe it's time to move this out into a variable instead of prettier making the formatting look like this?